### DISCUSSION TITLE
Websockets

### DISCUSSION THREAD

**Discussion by azriel46d** [usebruno/bruno#376] [Q&A] [answered]
Hi, first of all amazing project!
Are there any plans in the roadmap to support websockets

**Comment by helloanoop** [comment, marked as answer]
Yes, For V1 which we expect to wrap up very soon - key priority is to solve most of the major use cases for ReST api's
For V2, we will focus on Websockets and GRPC support

**Comment by azriel46d** [comment]
Might i suggest a roadmap or a rudimentary planned todo list? This might help even in terms of contributions to the code based on the planned roadmap :)

**Reply by helloanoop** [reply]
Yes. Will create one and link it in our docs, website and repo's

**Reply by helloanoop** [reply]
I have created our roadmap here: https://github.com/usebruno/bruno/issues/119
Have added links in our website, repo as well as docs.

**Reply by azriel46d** [reply]
Nice! Thank you :)
